integrations/lib/signals.go

```diff
@@ -41,6 +41,7 @@ func ServeSignals(app Terminable, shutdownTimeout time.Duration) {
 	signal.Notify(sigC,
 		syscall.SIGTERM, // graceful shutdown
 		syscall.SIGINT,  // graceful-then-fast shutdown
+		syscall.SIGUSR1,
```

Comment:
Can we please add some kind of comment explaining what it does? 
```suggestion
		syscall.SIGUSR1, // log process diagnostic info
```